docs/ThirdPartyHosting.md

```diff
@@ -18,7 +22,7 @@ Go to Server Settings > Policies > Disable registration, unmark the checkbox. Yo
 
 ## Are there any limitations in features when using a third party host?
 Yes. Here are some restrictions.
-* No [Lightning Network](LightningNetwork.md)
+* No [Lightning Network](LightningNetwork.md) by default. Can be enabled by the third-party host but at the cost of a security risk for registrants using that third-party. [More information](#Can-I-enable-the-use-of-my-Lightning-Network-node-to-others)
```

Comment:
Normally you can just use lower case for the words but I don't know if it matters. Either way I don't think the link works.
`can-i-enable-the-use-of-my-lightning-network-node-to-others` I believe this doesn't link to the section below because the question title contains an extra space at the end before it's '?'. I made the suggestion below.

Reply:
```suggestion
* No [Lightning Network](LightningNetwork.md) by default. Can be enabled by the third-party host but at the cost of a security risk for registrants using that third-party. [More information](#can-i-enable-the-use-of-my-lightning-network-node-to-others)
```

Lower-cased the words, just in case.